fitness_functions/publish.py

```diff
@@ -1,2 +1,61 @@
+import os
+import ast
+import sqlite3
+from datetime import datetime
+from itertools import islice
+import matplotlib.pyplot as plt
+from sklearn import preprocessing
+from fitness_functions.run import run
+
+
+# Function to normalise the graph data
+def normalise(fitness_dictionary):
+    for key, value in islice(fitness_dictionary.items(), 1, None):
+        fitness_dictionary[key] = [round(n * 100, 0) for n in preprocessing.normalize([value])[0]]
+    return fitness_dictionary
+
+
 def publish(project_path, code_path):
-    print('publish')
+    project_fitness_directory = os.path.join(project_path, 'fitness')
+
+    try:
+        connection = sqlite3.connect(os.path.join(project_fitness_directory, 'fitness_metrics.db'))
+        cur = connection.cursor()
+    except:
```

Comment:
sqlite3.connect() creates a database at the specified path if it doesn't exist. I would first manually check if the db exists, and if not, raise an exception alerting the user than they need to call run() before calling publish():

```suggestion
    if not path.exists(os.path.join(project_fitness_directory, 'fitness_metrics.db')):
        raise Exception('Please ...')
```